### PR TITLE
Feature/history api

### PR DESCRIPTION
**Pull request by DerMolly**

### CODE REVIEW

File: src/api/history.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+export const setHistory = async (entries: HistoryEntry[]): Promise<void> => {
+  const response = await fetch(getBackendUrl() + '/history', {
+    method: 'POST',
+    mode: 'cors',
+    cache: 'no-cache',
+    credentials: 'same-origin',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    redirect: 'follow',
```
Comment (ErikMichelson) [inline src/api/history.ts:19]: Why should we follow redirects?

Comment (mrdrogdrog) [reply src/api/history.ts:19]: Because.. of.. redirecting load balancers? 😅
No srsly. Why shouldn't we?

Comment (DerMolly) [reply src/api/history.ts:19]: To be honest I just copied the settings of the other api calls, maybe we should extract default fetch config

Comment (DerMolly) [reply src/api/history.ts]: extracted

**Pull request merged**